src/core/server/base.js

```diff
@@ -1,10 +1,9 @@
-/* eslint-disable no-console */
```

Comment:
I moved these to the specific lines since I had added a few debug statements from time to time and thought I might forget them.